docs/source/prebuilt_presets/helpers.rst

```diff
@@ -140,3 +140,39 @@ default. You can change this number by setting the override variable
 Once the entire channel is downloaded, remove the usage of this preset. It will then
 pull metadata from newest to oldest again, and stop once it reaches a video that has
 already been downloaded.
+
+
+_throttle_protection
```

Comment:
Maybe we should give throttle an entire page to itself. I already think 'Helpers' is a bit too vague. Wdyt?

Reply:
Yeah, I went around on this a few times before deciding on this placement. I think increasing the number of pages comes with its own downsides, muddying where to find things for both readers and editors. Where do I find the non-ref docs on throttle protection? Helpers makes sense. It's own page? Why this and not others? So yeah, having done another lap through this, I still think this is the best placement until this section grows significantly.